tests/framework/installer/ceph_installer.go

```diff
@@ -62,7 +62,9 @@ type CephInstaller struct {
 	hostPathToDelete string
 	helmHelper       *utils.HelmHelper
 	useHelm          bool
+	enableCSI        string
```

Comment:
Yep, I think lets enable it by default.

Reply:
we need some way to set [ROOK_CSI_ENABLE_CEPHFS](https://github.com/rook/rook/blob/master/tests/framework/installer/ceph_manifests.go#L871)  to `"false"`  to disable starting of cephCSI.
I thought its good to add a  new field to installer struct, Any better approach to fulfill this?
This is much useful in cephcsi

Reply:
In 1.1 we will want to have CSI enabled by default, so in the master branch we can start moving in that direction. So I think we should always enable CSI in the integration tests and there is no need for the new option. The option to enable CSI will still exist for now in the operator env vars, but the integration tests don't need that option, right?

Reply:
@travisn any other way to make use of rook integration tests without enabling the CSI,
so that we can use the integration framework for deploy rook and ceph cluster, not CSI.
current `StartTestCluster` start the rook operator with CSI.

is there a way to start only rook operator without CSI in integration tests?

Reply:
> Yep, lets enable CSI by default.

@humblec enabling CSI in rook-operator will pull the stable version of CSI and also it as its own yaml files for the deployment of CSI.

Reply:
@Madhu-1 you mean `1.1` as the stable version , Isnt it ?
@travisn I would like to enable CSI in rook as default soon after release of CSI v1.1 which is currently planned in a ~week time , may be not before that.  :)

Reply:
@humblec yes 1.1 will be CSI next stable version